etc/docker/docker_entrypoint.sh

```diff
@@ -81,7 +81,7 @@ trap shutdown TERM INT
 
 parse_args "$@"
 
-[ -n "$RADIO_URL" ] || RADIO_URL="spinel+hdlc+uart:///dev/ttyUSB0"
```

Comment:
there's another combination: `spinel+hdlc+forkpty`, right?

Reply:
Sorry I missed it. We need to keep it.